features/aanspreekvorm.feature

```diff
@@ -0,0 +1,659 @@
+      | aanduiding naamgebruik (61.10) | <aanduiding naamgebruik> |
+      En de persoon heeft een 'partner' met de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      | naam                                                               | waarde   |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) | 20180808 |
+      En de 'partner' heeft de volgende 'ontbindingHuwelijkPartnerschap' gegevens
+      | naam                                                         | waarde   |
+      | datum ontbinding huwelijk/geregistreerd partnerschap (07.10) | 20201001 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met een leeg 'adressering' object
+
+      Voorbeelden:
+      | aanduiding naamgebruik |
+      | E                      |
+      | P                      |
+      | V                      |
+      | N                      |
+
+  Rule: Er is geen aanspreekvorm als de persoon de naam van de (ex-)partner zonder eigen geslachtsnaam gebruikt (aanduiding naamgebruik is gelijk aan "P")
+
+    Abstract Scenario: persoon met geslacht "<geslacht>", adellijke titel en aanduiding naamgebruik ongelijk aan "P", heeft partner met/zonder adellijke titel/predicaat 
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde     |
+      | burgerservicenummer         | 999992934  |
+      | geslachtsaanduiding (04.10) | <geslacht> |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde            |
+      | adellijke titel of predicaat (02.20) | <adellijke titel> |
+      | aanduiding naamgebruik (61.10)       | <naamgebruik>     |
+      En de persoon heeft een 'partner' met de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      | naam                                                               | waarde   |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) | 20180808 |
+      En de 'partner' heeft de volgende 'naam' gegevens
+      | naam                                 | waarde                    |
+      | adellijke titel of predicaat (02.20) | <titel/predicaat partner> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                          | waarde          |
+      | aanschrijfwijze.aanspreekvorm | <aanspreekvorm> |
```

Comment:
ik zou de uitvalregels voor aanschrijfwijze.naam ook voor aanschrijfwijze.aanspreekvorm hanteren. Ik denk dat het niet nuttig is alleen de aanspreekvorm te leveren wanneer je niet ook de naam hebt.

Dus in dit geval heeft de persoon geen geslachtsnaam en ook de partner heeft geen geslachtsnaam en daarom verwacht ik geen aanschrijfwijze.